components/App.jsx

```diff
@@ -8,86 +8,88 @@ import Sprite from './Sprite';
 import AnimationPane from './AnimationPane';
 import SpeechBox from "./SpeechBox";
 import MainNav from "./MainNav";
+import AboutPage from './pages/AboutPage';
+import WorkPage from "./pages/WorkPage";
+import ContactPage from "./pages/ContactPage";
 import { Switch, BrowserRouter as Router, Route } from 'react-router-dom'
 
 //SPRITE SHIFTING FUNC
 //TODO: connect dialogue change to speechbox
 //TODO: add work page dialogue to array and get it working
 const App = () => {
+    // 
     const [spriteClicked, setSpriteClicked] = useState(false);
-    const [paneActive, setPaneActive] = useState(false);
-    const [isWorkActive, setIsWorkActive] = useState(false);
     const [isSpeechBoxActive, setIsSpeechBoxActive] = useState(false);
     const [isSpriteInSpeechBox, setIsSpriteInSpeechBox] = useState(false);
-    const [isNav, setIsNav] = useState(true);
-    let speechBoxData = dialogueObj.intro;
+    const [speechBoxData, isSpeechBoxData] = useState(dialogueObj.intro)
+    // I think this wants to be refactored to some other name
+    const [isNav, setIsNav] = useState(false);
+    // are these pages? things like panes take up this space
+    const [page, setPage] = useState('home')
+
     let initialSpriteClick = true;
 
+    // does this do way too much?
     const spriteClickHandler = () => {
-        if(initialSpriteClick){
+        if (initialSpriteClick) {
             setSpriteClicked(true);
-            setPaneActive(true);
-    
+            setPage('pane');
+            // setPaneActive(true);
+            // useEffect for animation change here too
             setTimeout(() => {
                 setIsSpeechBoxActive(true);
             }, 500);
+            // this else block never runs for some reason, need to figure that out
         } else {
-            speechBoxData = dialogueObj.work;
+            // check this works
+            setSpeechBoxData(dialogueObj.work);
             console.log(speechBoxData);
             setIsSpeechBoxActive(true);
         }
-        
-    }
 
-    const showNav = () => {
-        setIsNav(true);
     }
 
     const removeSpeechBox = () => {
         setIsSpeechBoxActive(false);
     }
 
-    const paneSlideUp = (pane) => {
-        gsap.to(pane, {duration: .7, scaleY: 1, transformOrigin: 'bottom'});
-    }
+    // I'm not sure this sounds like a function, could be state? 
+
 
-    const navLinkHandler = () => {
-        setIsWorkActive(true);
-        removeSpeechBox();
+    const pages = {
```

Comment:
Hey, I see you have a pages dictionary, this is really cool, but I'm not sure where you're using?